src/site/content/en/blog/houdini-how/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Cross-browser paint worklets and Houdini.how'
+subhead: CSS Houdini worklets and resources are just a few clicks away.
+authors:
+  - una
+date: 2020-12-10
+hero: hero.jpg
+alt: A sparkler.
+description: Learn how to implement cross-browser Houdini Paint API's and explore a world of Houdini worklets with Houdini.how.
+tags:
+  - blog
+  - css
+  - houdini
+---
+
+CSS [Houdini](https://developer.mozilla.org/en-US/docs/Web/Houdini) is an umbrella term that describes a series of low-level browser APIs that give developers much more control and power over the styles they write. 
```

Comment:
Please break all lines at 100 characters. If you're using VS Code, [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) makes it pretty easy to do. The [Docs To X](https://gsuite.google.com/u/0/marketplace/app/docs_to_x/243920501110?hl=en&pann=docs_addon_widget) Google Docs Add-On also supports this.